apps/web/src/components/AvatarDropzone/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
```

Comment:
Follow the convention please
```
index.ts
AvatarDropzone.tsx
styles.ts
```

Create components with the `yarn plop` next time, it will do all of this for you.

Reply:
done